.eslintrc.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
       { avoidEscape: true, allowTemplateLiterals: false }
     ],
     curly: ['error', 'all'],
-    eqeqeq: 'error',
+    eqeqeq: ['error', 'always', { null: 'ignore' }],
```

Comment:
Do we still need the `null: 'ignore'` part? Or was is for testing?

It would be nice to stick to `===` checks in general, but maybe this was added for a specific reason?

Reply:
strict equality of `null` and `undefined` is not consistent, whereas if you let the browsers Do Their Casting Thing, it comes out correctly....

Reply:
So do you think we should keep that `ignore` clause, or leave it like it was before?

It might sound like a cosmetic thing mostly, but I was thinking we could try to follow the [JupyterLab TypeScript Style Guide](https://github.com/jupyterlab/jupyterlab/wiki/TypeScript-Style-Guide) to make it easier for folks to switch between code bases and contribute to these projects. It doesn't explicitly say anything about strict equality, but I guess it would try to follow the JupyterLab eslint / prettier config when that makes sense.